src/main/scala/com/monsanto/arch/cloudformation/model/resource/ElastiCache.scala

```diff
@@ -0,0 +1,64 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model.{ConditionRef, ResourceRef, Token}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+
+/**
+  * Cache security group.
+  *
+  * @param name
+  * @param Description
+  * @param SubnetIds
+  * @param Condition
+  */
+case class `AWS::ElastiCache::SubnetGroup`(
+  name:                     String,
+  Description:              String,
+  SubnetIds:                Token[Seq[ResourceRef[`AWS::EC2::Subnet`]]],
+  override val Condition:    Option[ConditionRef] = None
+  ) extends Resource[`AWS::ElastiCache::SubnetGroup`]{
+
+  def when(newCondition: Option[ConditionRef] = Condition) = copy(Condition = newCondition)
+}
+
+/**
+  * Json format definition for subnet group.
+  */
+object `AWS::ElastiCache::SubnetGroup` extends DefaultJsonProtocol {
+  implicit val format: JsonFormat[`AWS::ElastiCache::SubnetGroup`] = jsonFormat4(`AWS::ElastiCache::SubnetGroup`.apply)
+}
+
+/**
+  * Cache cluster definition.
+  *
+  * @param name
+  * @param ClusterName
+  * @param CacheNodeType
+  * @param CacheSubnetGroupName
+  * @param VpcSecurityGroupIds
+  * @param Engine
+  * @param NumCacheNodes
+  * @param Tags
+  * @param Condition
+  */
+case class `AWS::ElastiCache::CacheCluster`(
```

Comment:
again  there are additional parameters we could add in here [docs](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticache-cache-cluster.html)